README.md

```diff
@@ -560,6 +560,7 @@ There are [other sites with curated lists of elixir packages](#other-awesome-lis
 
 * [bake](https://github.com/bakeware/bake) - Configure, compile and share systems, toolchains and linux firmware.
```

Comment:
Hm... we should probably remove `bakeware` while you're in here. 😬 
That hasn't been relevant for quite some time.

Reply:
https://github.com/h4cc/awesome-elixir/pull/4523